modules/allowances/package.json

```diff
@@ -17,17 +17,25 @@
     "prepare": "pnpm run clean && npm run build",
     "lint": "pnpm run lint:sol && npm run lint:ts",
     "lint:sol": "solhint 'contracts/**/*.sol'",
-    "lint:ts": "eslint 'tasks/**/*.ts' 'test/**/*.ts' --max-warnings 0 --fix",
-    "fmt": "pnpm run fmt:sol && npm run fmt:ts",
-    "fmt:sol": "prettier 'contracts/**/*.sol' -w",
-    "fmt:ts": "prettier 'tasks/**/*.ts' 'test/**/*.ts' -w"
+    "lint:sol:fix": "solhint 'contracts/**/*.sol' --fix",
+    "lint:ts": "eslint 'tasks/**/*.ts' 'test/**/*.ts' --max-warnings 0",
+    "lint:ts:fix": "eslint 'tasks/**/*.ts' 'test/**/*.ts' --max-warnings 0 --fix",
+    "fmt": "pnpm run fmt:sol:write && npm run fmt:ts:write",
+    "fmt:check": "pnpm run fmt:sol:check && npm run fmt:ts:check",
+    "fmt:sol:check": "prettier 'contracts/**/*.sol' --check",
+    "fmt:ts:check": "prettier 'tasks/**/*.ts' 'test/**/*.ts' --check",
+    "fmt:sol:write": "prettier 'contracts/**/*.sol' --write",
+    "fmt:ts:write": "prettier 'tasks/**/*.ts' 'test/**/*.ts' --write"
   },
-  "author": "richard@gnosis.io",
-  "license": "ISC",
+  "author": "safe-global",
+  "license": "GPL-3.0",
```

Comment:
FYI: GPL-3.0 is deprecated. https://spdx.org/licenses/GPL-3.0.html

Reply:
This is the one we use in the other files, I'd raise this with legal. It says the identifier has been deprecated but not the license. We need a translation from legalese

Reply:
I changed it to GPL because this is the one we use in other files. I see it's deprecated and asked in Slack which one we should use.

Let's address this later in a different PR (I can also revert the change until we get more clarity)

Reply:
> This is the one we use in the other files, I'd raise this with legal. It says the identifier has been deprecated but not the license. We need a translation from legalese

The non-deprecated identifier is `GPL-3.0-only`.

Reply:
Basically, the `GPL-3.0` identifier was ambiguous because GPL has a "license update clause" (meaning that if they come up with a GPL-4, then all GPL-3 code will also be licensed under v4). In SPDX, this was represented by `GPL-3.0-or-later`. `GPL-3.0` changed to `GPL-3.0-only` to avoid confusion (as it is the GPL license opting out of the "license update clause").

Reply:
Looking at the contract code, this should be `LGPL-3.0-only` (see the SPDX identifier used in the Solidity code).